reference/v8.json

```diff
@@ -1241,43 +1359,64 @@
   },
   "filter_operator": {
     "type": "enum",
-    "values": [
-      "==",
-      "!=",
-      ">",
-      ">=",
-      "<",
-      "<=",
-      "in",
-      "!in",
-      "all",
-      "any",
-      "none",
-      "has",
-      "!has"
-    ],
+    "values": {
+      "==": {
+          "doc": "The equal-to operator."
```

Comment:
Yes, that's what I meant by "pull in the descriptions from https://www.mapbox.com/mapbox-gl-style-spec/#types-filter". I think it would also be good to set up the `key` and `value` variables as the existing documentation does:

![image](https://cloud.githubusercontent.com/assets/98601/19003275/edd36be6-8704-11e6-8fa4-dcdf3f8ac0bc.png)